packages/admin/cms-admin/src/common/header/impersonation/ImpersonationInlay.tsx

```diff
@@ -0,0 +1,48 @@
+import { Reset } from "@comet/admin-icons";
+import { Box, Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { FormattedMessage } from "react-intl";
+
+import { useCurrentUser } from "../../../userPermissions/hooks/currentUser";
+import { StopImpersonationButton } from "../../../userPermissions/user/ImpersonationButtons";
+
+export function ImpersonationInlay() {
+    const user = useCurrentUser();
+
+    return (
+        <Inlay padding={4}>
+            <Tag>
+                <FormattedMessage id="comet.impersonation.inlay.title" defaultMessage="Impersonation mode" />
+            </Tag>
+            <Typography variant="h6" sx={{ marginTop: 3 }}>
+                {user.name}
+            </Typography>
+            <Typography variant="body2" sx={{ marginBottom: 3 }}>
+                {user.email}
+            </Typography>
+            <StopImpersonationButton startIcon={<Reset />} fullWidth variant="contained" color="secondary" sx={{ justifyContent: "center" }} />
```

Comment:
I believe the text of this button needs to be updated. It's currently "Stop Impersonation"